backend/app/tests/api/routes/scheduled_inquiry/test_add_to_schedule_api.py

```diff
@@ -0,0 +1,43 @@
+from fastapi.testclient import TestClient
+from sqlmodel import Session
+
+from app.models import InquiryCreate
+from app.services import inquiries as inquiries_service
+from app.tests.utils.api_post import api_post
+
+route_prefix = "scheduledinquiries"
+
+
```

Comment:
Is there a DB clean up for after running API layer tests?

There's a cleanup fixture inside `confest.py` that we've been using to specify tables we wanted to cleanup after  running each test:
```
# Clear specified tables after running each test function
@pytest.fixture(scope="function", autouse=True)
def clear_tables_after_tests(db: Session) -> Generator[None, None, None]:
    yield
    tables_to_clear = [Inquiry]
    for table in tables_to_clear:
        statement = select(table)
        results = db.exec(statement).all()
        for record in results:
            db.delete(record)
    db.commit()
```

Reply:
Thats part of the reason why I added the ondelete=CASCADE relationship between scheduled inquiries and inquiries. When this fixture runs it deletes the inquiries and scheduled inquiries by cascade. I'm not sure I love this fixture implementation. When you're dealing with relationships order matters when your deleting from the db. Since its doing a linear iteration here you have to make sure everything is in the right order for it to work. I'm not sure how well this will scale

Reply:
Got it. Yeah, I think this fixture function is not useful to scale. Ideally we want to automatically clear the tables that changed from each test. We could revisit this in future PR.